src/lib/bn.c

```diff
@@ -181,7 +181,7 @@ PGPV_BN_div(PGPV_BIGNUM *      dv,
             const PGPV_BIGNUM *d,
             PGPV_BN_CTX *      ctx)
 {
-    if ((dv == NULL && rem == NULL) || a == NULL || d == NULL) {
+    if ( (dv == NULL) || (rem == NULL) || (a == NULL) || (d == NULL)) {
```

Comment:
@flowher it is a very good changes because few lines below I see
`botan_mp_div(dv->mp, rem->mp, a->mp, d->mp) == 0;`

So, if dv or rem is NULL it's crashed with SEGFAULT but I haven't see any code at whole rnp where this function calls